statsd-client-async/statsd_client_async.ml

```diff
@@ -33,13 +33,19 @@ module T =  struct
     | Unix.ADDR_UNIX _ -> failwith "Unix domain addresses not supported"
 
   let sendto fd msg _offset msg_length _flags socket_address =
-    match Udp.sendto () with
+    match Async_udp.sendto_sync () with
```

Comment:
so async has a function thats sync? interesting...

Does this fix (or continue to resolve) the memory leak issue?

Reply:
That commit is something @vgrocha added a while ago and we're already using it. I just wanted to merge it into master since pinning this branch is confusing.